### ISSUE TITLE
How can I add one new class to MS COCO dataset and do transfer learning to achieve more classes? Already, I collect images with new class, annotate them and create Tfrecord but the result of transfer learning doesn't recognize that new class!

### ISSUE THREAD

**Issue by mojgd** [tensorflow/models#6617]
I've annotated new image with new class according to the approach is recommended by http://www.immersivelimit.com/tutorials/create-coco-annotations-from-scratch

then I combined new class annotations with coco json annotate files and create TFrecord by running 
create_coco_tf_record.py from this repository : https://github.com/tensorflow/models/tree/master/research/object_detection/dataset_tools

then I retrained faster_rcnn_resnet101 which its pretrained model is available in Tensorflow Model Zoo. I did retrain process according to this tutorial (from step5 to 8) : https://github.com/EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10

as you can see bellow, the loss curve didn't converge and also had oscillation!! 

how can I do to get better result from retraining models?
![lossCurve](https://user-images.githubusercontent.com/46745567/56453175-69cd2c00-6353-11e9-825b-e400befad0b4.PNG)
![lossCurve-200K](https://user-images.githubusercontent.com/46745567/56453176-69cd2c00-6353-11e9-9399-8a0260c4d51d.PNG)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
